external/storm-kafka/src/jvm/storm/kafka/KafkaUtils.java

```diff
@@ -82,11 +82,9 @@ public static long getOffset(SimpleConsumer consumer, String topic, int partitio
     public static class KafkaOffsetMetric implements IMetric {
         Map<Partition, Long> _partitionToOffset = new HashMap<Partition, Long>();
         Set<Partition> _partitions;
-        String _topic;
         DynamicPartitionConnections _connections;
 
-        public KafkaOffsetMetric(String topic, DynamicPartitionConnections connections) {
-            _topic = topic;
```

Comment:
any reason we have to delete this? I am thinking about the case where some users will have graphs setup on per topic bases and it will be broken after this change.